### ISSUE TITLE
Question about experimental result differs from original paper

### ISSUE THREAD

**Issue by lizijian-buaa** [3DAgentWorld/Toolkit-for-Prompt-Compression#2]
Hi. In your paper's Table 3, for Arxiv dataset, Selective Context gets higher BLUE score than LLMLingua. But in LLMLingua paper, in all settings, LLMLingua is better than Selective Context, using BLUE score as the indicator.

**Comment by lizijian-buaa** [comment]
BTW, from my own reproduction, Selective Context also gets higher BLUE score for Arxiv dataset.

**Comment by Leikrit** [comment]
Hello, thanks for asking. This issue is mainly because of two aspects: firstly, in LLMLingua, they used completion mode, which is not available recently, so we used chat mode (it is also mentioned in our paper), completion mode is more suitable for some structural tasks while chat mode is more suitable for pure text tasks. Another factor that causes the difference is the instructions used in each task. We tested different instructions (how you describe the task to GPT) and found that for some tasks, LLMs perform sensitively to the instructions. However, I have not yet found the original instruction that is used in LLMLingua's experiments, so it may cause the difference in the experiment results as well.
